### PR TITLE
docs: add automation examples

### PR DESCRIPTION
**Pull request by Cafe137**
[Preview the README](https://github.com/ethersphere/swarm-cli/blob/98dcb2cf0ad806d6e38653c75dad0d6f8c13f182/README.md)

- Fixes writing errors to stderr
- Sets exit code to `1` when `status` has failures
- Changes description tagline to match Swarm-CLI tagline
- Adds Automation section 
- Adds Table of Contents

### CODE REVIEW

**Comment by Cafe137** [comment]
@agazso @nugaon I am curious what you think about adding the Table of Contents. 

The README is quite long, I think it would be beneficial to break it up to smaller parts and/or utilise the wiki features.

But for now, adding a ToC gives a nicer overview of the topics which partially solves this problem.

File: README.md [hunk 1]
```diff
@@ -9,9 +9,37 @@
...
+      * [From source](#from-source)
+   * [Usage](#usage)
+   * [Commands](#commands)
+   * [Example usage](#example-usage)
+   * [Usability Features](#usability-features)
+      * [Numerical Separator and Units](#numerical-separator-and-units)
+      * [Stamp Picker](#stamp-picker)
+      * [Identity Picker](#identity-picker)
+      * [Human Readable Topics](#human-readable-topics)
+      * [Automatizing tasks with Swarm-CLI](#automatizing-tasks-with-swarm-cli)
```
Comment (agazso) [inline README.md:28]:
```suggestion
      * [Automating tasks with Swarm-CLI](#automating-tasks-with-swarm-cli)
```

**Comment by agazso** [comment]
> @agazso @nugaon I am curious what you think about adding the Table of Contents.
> 
> The README is quite long, I think it would be beneficial to break it up to smaller parts and/or utilise the Wiki feature.
> 
> But for now, adding a ToC gives a nicer overview of the topics which partially solves this problem.

I think the ToC is very nice. I have seen projects with even bigger ToC and I don't think that's a problem. In the case you would want to break it up into smaller parts I would suggest to put the documentation in another file (e.g. DOCUMENTATION.md) instead of using the wiki feature, because that way the cloned repo will also have access to the documentation.

**Comment by Cafe137** [comment]
Thanks for the feedback @agazso, I agree the README structure and size is good for now.

What do you think of the examples and the writing? Is this good enough, or should we expand on it more?

**Review by agazso — COMMENTED** [review]
`status` command seems to be missing from the Swarm-cli output `Available commands` section.

**Comment by agazso** [comment]
> Thanks for the feedback @agazso, I agree the README structure and size is good for now.
> 
> What do you think of the examples and the writing? Is this good enough, or should we expand on it more?
> 
> Edit: I know we discussed an _"automation vs interactive usage"_ section, but it felt out of the place since there are already 2 sections about interactiveness, so I ended up creating one for the automation on its own.

I think this is an improvement already. The examples are useful, but I would be a bit cautious about using them because if we change the output of the (for example) stamp listing they will break but there is no automatic way to test those. So maybe it would be useful to provide examples that are less prone to breaking.

File: README.md [hunk 2]
```diff
@@ -9,9 +9,37 @@
...
+   * [System environment](#system-environment)
+* [Development](#development)
+* [Contribute](#contribute)
+* [Maintainers](#maintainers)
+* [License](#license)
+
 # Description
 
-> Command line interface tool for manage Bee node and utilize its functionalities
+> Manage your Bee node and interact with the Swarm network via the CLI
```
Comment (agazso) [inline README.md:42]: Nice! After the merge let's change the about message of the repo to this.

**Review by agazso — APPROVED** [review]

**Pull request merged**